var/spack/repos/builtin/packages/sensei/package.py

```diff
@@ -55,7 +55,6 @@ class Sensei(CMakePackage):
     depends_on("paraview+mpi", when="+catalyst")
     depends_on("paraview+hdf5", when="+catalyst+hdf5")
     depends_on("paraview+python", when="@:2 +catalyst+python")
```

Comment:
Remove the version constraint here or change it to `@3:`

Reply:
Done